features/_your_contributions/percent_validation.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Informing about contribution percentages
+  In order to gain a better understanding of how my workplace pension is being calculated,
+  As a user of the WPCC tool,
+  I want to see relevant information about the contributions my employer and I can make.
+
+Background:
+  Given I am on the Your Details step
+  When I fill in my details
+
+Scenario Outline: minimum contribution percentage on salary greater than £5,876
+  And I proceed to the next step
+  And the "employee" contribution intro should display "<message>"
```

Comment:
We don't really need to quote `"employee"` or `"employer"` as it's not been used as part of the examples in our scenario outline.

Reply:
The quotes aren't for use with the examples table. That is just used to pass an argument (`contributor`) to the step definition to determine which part of the page it should be looking 
https://github.com/moneyadviceservice/wpcc/pull/146/files#diff-28bfa4aea260422424fcf9612be5d0d6R169
```
Then(/^the "([^"]*)" contribution intro should display "([^"]*)"$/) do |contributor, message|
  within(".contributions__source--#{contributor}") do
    ...
```

Reply:
it is only arguments with `<...>` that get passed to the examples table

Reply:
Ok, 👍 that looks good.